main.cf

```diff
@@ -34,7 +34,7 @@ smtp_tls_session_cache_database = btree:${data_directory}/smtp_scache
 # information on enabling SSL in the smtp client.
```

Comment:
We want this file to be gone and taken directly from resource directory from deployment project

Reply:
done